packages/serverless-orchestration/src/ServerlessHub.js

```diff
@@ -592,7 +614,14 @@ if (require.main === module) {
     process.exit(1);
   }
 
-  Poll(null, process.env.PORT, process.env.SPOKE_URL, process.env.CUSTOM_NODE_URL, hubConfig).then(() => {});
+  Poll(
```

Comment:
it might be cleaner to changing this to have the `hubConfig` still at the end (i.e order like this):
```
    null,
    process.env.PORT,
    process.env.SPOKE_URL,
    process.env.CUSTOM_NODE_URL,
    process.env.SPOKE_URLS
    hubConfig,
```

Reply:
this will end up breaking some tests, but i can update them